src/content/docs/developer-tools/kinde-api/access-token-for-api.mdx

```diff
@@ -12,27 +13,15 @@ app_context:
     s: applications
 ---
 
-To securely connect to Kinde’s API, you need to obtain an access token. You also need to have:
+To securely connect to Kinde’s API, you need to obtain an access token. Once you have completed the [connection steps](/developer-tools/kinde-api/connect-to-kinde-api/), you're ready to make an access token request.
```

Comment:
## Prerequisites

* [Create and authorize an M2M application for the Kinde management API](developer-tools/kinde-api/connect-to-kinde-api/)

## Get access tokens

To call the [Kinde Management API](https://kinde.com/api/docs/) endpoints you need to authenticate with an access token. These tokens are JSON Web Tokens (JWTs) which contain specific permissions known as [scopes](/developer-tools/kinde-api/about-m2m-scopes).

To ask Kinde for an access token for calling the management API, perform a `POST` request to the `https://<your_subdomain>.kinde.com/oauth2/token` endpoint, using the credentials of the Machine-to-Machine application you created in the prerequisite step.

The payload should look as follows:
```
curl --request POST \
  --url 'https://<your_subdomain>.kinde.com/oauth2/token' \
  --header 'content-type: application/x-www-form-urlencoded' \
  --data grant_type=client_credentials \
  --data 'client_id=<your_m2m_client_id>' \
  --data 'client_secret=<your_m2m_client_secret> \
  --data 'audience=https://<your_subdomain>}.kinde.com/api'
  ```

You can call the token endpoint in the backend language of your choice, for example in NodeJS this would look like:
```js
fetch(`https://<your_subdomain>.kinde.com/oauth2/token`, {
	method: "POST",
	headers: {
		"content-type": "application/x-www-form-urlencoded",
	},
	body: new URLSearchParams({
		audience: "https://<your_subdomain>.kinde.com/api",
		grant_type: "client_credentials",
		client_id: <your_m2m_client_id>,
		client_secret: <your_m2m_client_secret>
	})
});
```

The response will contain a signed JWT  containing claims including the `scope`s the token is allowed to access and the expiry time. Here is an example token:
```
{
  "aud": [
    "https://example.kinde.com/api"
  ],
  "azp": "bd69bb9fe5db44a38b6b2dacd1f4b451",
  "exp": 1729812040,
  "gty": [
    "client_credentials"
  ],
  "iat": 1729725640,
  "iss": "https://example.kinde.com",
  "jti": "6f091ebe-44ba-4afc-bd2f-05fcccafc89e",
  "scope": "read:users update:users",
}
```

## Use the access token

To use this token, include it in the Authorization header of your request. For example to [get all users](https://kinde.com/api/docs/#get-users) you would call:
```
curl --request GET \
  --url 'https://<your_subdomain>.kinde.com/api/v1/users' \
  --header 'authorization: Bearer <m2m_access_token>' \
  --header 'content-type: application/json'
 ```

## Use the Kinde Management API JS SDK

As an alternative to making HTTP calls, you can also use the [Kinde Management API JS](https://github.com/kinde-oss/management-api-js) SDK.

You can use it to automatically obtain tokens for, and interact with the Kinde management API.